### PR TITLE
Trial run of removing/transforming qplot examples in ggplot2 documentation 

### PR DESCRIPTION
**Pull request by hrbrmstr**
Changed `qplot` examples to regular ggplot calls in the documentation of 5 source files and also made the spaces around "=" uniform in the examples in those files. I really thought about changing `plyr` usage into `dplyr` calls at the same time but I thought it best to keep the edits focused. If this ends up working OK, I'll make quick work of the rest of it.

### CODE REVIEW

File: R/aes-linetype-size-shape.r [hunk 1]
```diff
@@ -31,7 +31,7 @@
 #'   date = economics$date,
 #'   colwise(rescale01)(economics[, -(1:2)]))
 #' ecm <- melt(ec_scaled, id = "date")
-#' qplot(date, value, data = ecm, geom = "line", linetype = variable)
+#' ggplot(ecm, aes(x= date, y = value)) + geom_line(aes(linetype = variable))
```
Comment (hadley) [inline R/aes-linetype-size-shape.r]: I think I now prefer one component on each line

File: R/annotation-custom.r [hunk 2]
```diff
@@ -22,7 +22,8 @@ NULL
 #' Inf values can be used to fill the full plot panel (see examples).
 #' @examples
 #' # Dummy plot
-#' base <- qplot(1:10, 1:10, geom = "blank") + theme_bw()
+#' df <- data.frame(x = 1:10, y = 1:10)
+#' base <- ggplot(df, aes(x = x, y = y)) + geom_blank() + theme_bw()
```
Comment (hadley) [inline R/annotation-custom.r]: I prefer not to use explicit x and y in aes

**Comment by hadley** [comment]
Looks good. This does seem like a good opportunity to clean up the style in the examples

**Comment by hrbrmstr** [comment]
I'll make those changes to those 5 and use the style moving forward.

**Comment by hadley** [comment]
Are you still interested in doing this? I'm in a good place to review and merge PRs at the moment (and should be for ~2 weeks)

**Comment by hrbrmstr** [comment]
Roger that. I have (IIRC) 3 functions left.

**Comment by hrbrmstr** [comment]
There are only a couple left. I'll try to get to them today.

**Pull request closed without merging**